Sources/WordPress/Info.plist

```diff
@@ -191,13 +192,13 @@
 				<key>UIPrerenderedIcon</key>
 				<true/>
 			</dict>
-			<key>Spectrum &apos;22</key>
+			<key>Spectrum '22</key>
 			<dict>
 				<key>CFBundleIconFiles</key>
 				<array>
-					<string>spectrum-&apos;22-icon-app-60x60</string>
-					<string>spectrum-&apos;22-icon-app-76x76</string>
-					<string>spectrum-&apos;22-icon-app-83.5x83.5</string>
+					<string>spectrum-'22-icon-app-60x60</string>
+					<string>spectrum-'22-icon-app-76x76</string>
+					<string>spectrum-'22-icon-app-83.5x83.5</string>
```

Comment:
Xcode made these changes. Xcode also made the original changes. Somehow, Xcode decides to revert changes that were made by itself...